004_team_2_agent_design/004_team_2_agent_design.tex

```diff
@@ -145,3 +234,239 @@ \subsubsection{Design of reward function}
     \end{itemize}
 \end{itemize}
 
+When tuning the hyper parameters in the reward function appropriately, the main factors for performance tuning were cumulative total tower death count, average agent life span, and number of Team 2 agent deaths. 
+
+\begin{table}[]
+\begin{adjustbox}{width=\columnwidth,center}
+\begin{tabular}{@{}rrrrrrr@{}}
+\toprule
+\multicolumn{1}{l}{Days} & \multicolumn{1}{l}{Overeating threshold} & \multicolumn{1}{l}{Overeating} & \multicolumn{1}{l}{Critical} & \multicolumn{1}{l}{Total Death:} & \multicolumn{1}{l}{Team2 Death per agent} & \multicolumn{1}{l}{Mean Age} \\ \midrule
+5000                     & 80                                       & 0.2                            & 10                           & 786                              & 8                                         & 88.14                        \\
+5000                     & 80                                       & 0.2                            & 5                            & 863                              & 7                                         & 72.94                        \\
+5000                     & 80                                       & 0.2                            & 1.5                          & 876                              & 10                                        & 63.44                        \\
+5000                     & 80                                       & 0.2                            & 3                            & 844                              & 7                                         & 73.11                        \\
+5000                     & 80                                       & 0.2                            & 15                           & 913                              & 6                                         & 71.63                        \\
+5000                     & 70                                       & 0.2                            & 10                           & 797                              & 12                                        & 71.61                        \\
+5000                     & 60                                       & 0.2                            & 10                           & 830                              & 13                                        & 82.71                        \\
+5000                     & 80                                       & 0.5                            & 10                           & 859                              & 9                                         & 80.48                        \\
+5000                     & 80                                       & 1                              & 1                            & 865                              & 6                                         & 69.33                        \\ \bottomrule
+\end{tabular}
+\end{adjustbox}
+\end{table}
```

Comment:
I think there should be a label and caption for this table to be referenced in the text.

Reply:
Added